### DISCUSSION TITLE
maxFacetsPerValue

### DISCUSSION THREAD

**Discussion by agwisniewska** [algolia/instantsearch#5525] [Q&A]
Hi, 

the docs say, that the `maxFacetsPerValue` limit is `1000`. I set a limit of 200 (because my facet's values are country_iso, so it's around 200), but as you see, even in the dashboard, only 100 are available. The same is true on the UI side, I cannot display more. Did I miss anything? 

![Screenshot 2023-03-02 at 13 34 48](https://user-images.githubusercontent.com/16926778/222672734-c846e46a-5296-47a9-bc48-e7568a22a13f.png)

![Screenshot 2023-03-02 at 13 34 57](https://user-images.githubusercontent.com/16926778/222672738-dc329974-5d3f-4106-addf-3323c6aae2b5.png)
 
![Screenshot 2023-03-02 at 13 34 23](https://user-images.githubusercontent.com/16926778/222672731-20f32f34-76cf-4fac-8585-4336107256a3.png)

**Comment by Haroenv** [comment]
The dashboard value is the default value of maxFacetValuesPerFacet, are you passing it in the UI as well with a `limit` or `maxValuesPerFacet` with a high value?
